<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DLT</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btAdd">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsX">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>450</y>
      <width>69</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsY">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>69</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="btClear">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>450</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>450</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNaive">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Naive Projective 4 pts</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ptOut">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>570</y>
      <width>521</width>
      <height>171</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbOut">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>550</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btDLT">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>500</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>DLT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNormalizedDLT">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>500</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Normalized DLT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbInX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbInY">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse input:</string>
    </property>
   </widget>
   <widget class="QSlider" name="vsScale">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>580</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btTransform">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>780</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Transform</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbNewPoint1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>750</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbNewPoint2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>810</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btNewPoint">
    <property name="geometry">
     <rect>
      <x>308</x>
      <y>780</y>
      <width>180</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Project New Point</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btUndo">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>450</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
